src/index.ts

```diff
@@ -134,6 +134,10 @@ program
     'Arguments to be passed to constructor of the program.',
     undefined,
   )
+  .option(
+    '--account <account>',
```

Comment:
Deploy has an account option but no wallet option, is this intentional? My understanding was that deploying does not use a wallet or account

Reply:
Ah, yes